cli/ssh_test.go

```diff
@@ -368,7 +368,7 @@ func TestSSH(t *testing.T) {
 		// Ensure that SSH_AUTH_SOCK is set.
 		// Linux: /tmp/auth-agent3167016167/listener.sock
 		// macOS: /var/folders/ng/m1q0wft14hj0t3rtjxrdnzsr0000gn/T/auth-agent3245553419/listener.sock
-		pty.WriteLine("env")
+		pty.WriteLine(`grep --perl-regexp --binary-files=text 'SSH_AUTH_SOCK=[^\x00]+' /proc/self/environ`)
```

Comment:
This won't work on macOS due to `/proc` and `--perl-regexp`. I'd suggest just doing `env | grep SSH_AUTH_SOCK=` for the best portability (note no requirement on char after `=` so as to show empty values as well.

Reply:
Updated!